tic-tac-toe-app/Square.jsx

```diff
@@ -0,0 +1,5 @@
+function Squares(props) {
+    return (
+     <div className="symbol" onClick={props.click}>{props.value}</div>
```

Comment:
This is just a matter of preference: when using multiple props, the [destructuring assignment](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Destructuring_assignment#Object_destructuring) can be used to help reduce code repetition. This is especially helpful in components with an increased number of props.
```suggestion
     const {click, value} = props;
     return (
     <div className="symbol" onClick={click}>{value}</div>
     )
```